spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -545,8 +544,6 @@ public void testPartialProgressEnabled() {
     Table table = createTable(20);
     int fileSize = averageFileSize(table);
 
-    table.updateProperties().set(COMMIT_NUM_RETRIES, "10").commit();
```

Comment:
Previously we need to increase the commit retries to pass the UT. Now we don't need this.